Converter/CMakeLists.txt

```diff
@@ -28,14 +26,22 @@ set(${PROJECT_NAME}_SRCS
 
 set(${PROJECT_NAME}_HDRS
   igtlioBaseConverter.h
-  #igtlioVideoConverter.h
   igtlioImageConverter.h
   igtlioPolyDataConverter.h
   igtlioStatusConverter.h
   igtlioTransformConverter.h
   igtlioStringConverter.h
   )
 
+IF(USE_H264 OR USE_VP9 OR USE_X265 OR USE_OpenHEVC)
```

Comment:
Perhaps, an aggregate variable (OpenIGTLink_Use_VideoStreaming or something like that) in the CMake code as well, as suggested by Adam for the c++ code. Furthermore, I would call variables for individual codecs OPENIGTLink_USE_H264 to make it clear those are defined by OpenIGTLink and not IO.